pkg/repositories/transformers/node_execution.go

```diff
@@ -118,6 +118,7 @@ func CreateNodeExecutionModel(ctx context.Context, input ToNodeExecutionModelInp
 		Phase:     input.Request.Event.Phase,
 		CreatedAt: input.Request.Event.OccurredAt,
 		UpdatedAt: input.Request.Event.OccurredAt,
+		DeckUri:   input.Request.Event.DeckUri,
```

Comment:
is the deck URI always written on every event? do we need to update `UpdateNodeExecutionModel` too?

Reply:
we did it already. [addTerminalState](https://github.com/flyteorg/flyteadmin/blob/master/pkg/repositories/transformers/node_execution.go#L98) in UpdateNodeExecutionModel

Reply:
> is the deck URI always written on every event

Only when the task succeeds.